### DISCUSSION TITLE
Wind modelling-mesh sensitivity

### DISCUSSION THREAD

**Discussion by mpachera** [firemodels/fds#11797] [Q&A] [answered]
Dear,

I would like to run a wind analysis in FDS and I found some guidelines about the mesh sizing and the domain size. Nevertheless I would like to make a small sensitivity analysis to check the impact of the mesh resolution near the building of interest. 
For me it's not feasible to simply refine the whole domain and rerun the calculation, due to computational limitations therefore I would like to study maybe a portion of the domain close to the zone of interest. 
Is there a minimum domain size I should simulate when using the wind method 1 discussed in the FDS UG or is this case dependent?

Thanks for the help
 
Matteo

**Comment by rmcdermo** [comment, marked as answer]
As is often the case, it depends on what you are asking of the model.  A
good test case to look at is the fds/Validation/Casara_Arts_Ribbed_Channel
case.  The results are actually shown in the FDS Verification Guide (search
on ribbed_channel).  There you will see the effect of resolution around the
block on the mean and rms velocity components, both normal to the wall and
along the wall.

Another good set of cases is the UWO wind tunnel cases for pressure
coefficient on a building.  These are in fds/Validation/UWO_Wind_Tunnel
